src/PhpDoc/ResolvedPhpDocBlock.php

```diff
@@ -129,6 +133,7 @@ public static function createEmpty(): self
 		$self->isDeprecated = false;
 		$self->isInternal = false;
 		$self->isFinal = false;
+		$self->friends = [];
```

Comment:
This property should be mentioned in `merge` method too.

Reply:
I assume it should also read `$result->friends = []` as friendship is not inherited? That's what I'm going with now

Reply:
Or prolly not, added some cases to the test and turns out this way it should be inherited :)